Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -39,21 +31,21 @@ import BraintreeCore
     /// Initializes a PayPal Vault request
     /// - Parameters:
     ///   - offerCredit: Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`.
+    ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
     ///   - recurringBillingDetails: Optional: Recurring billing product details.
     ///   - recurringBillingPlanType: Optional: Recurring billing plan type, or charge pattern.
-    ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
     public init(
         offerCredit: Bool = false,
         recurringBillingDetails: BTPayPalRecurringBillingDetails? = nil,
         recurringBillingPlanType: BTPayPalRecurringBillingPlanType? = nil,
         userAuthenticationEmail: String? = nil
     ) {
-        self.recurringBillingDetails = recurringBillingDetails
-        self.recurringBillingPlanType = recurringBillingPlanType
         super.init(
             offerCredit: offerCredit,
             userAuthenticationEmail: userAuthenticationEmail
         )
+        self.recurringBillingDetails = recurringBillingDetails
+        self.recurringBillingPlanType = recurringBillingPlanType
```

Comment:
i think we can pass these in the `super.init` like we do for `offerCredit` since these properties are now part of `BTPayPalRequest`